[#166] Bump javafx to version 11.0.2

Eclipse-Dominator opened the pull request:
Fixes #166 

Conflict between JavaFx version 11 and Ubuntu 18.04 and above described in JDK-8210411 presents a compatibility issue for users using Ubuntu 18.04 and above. This fatal crash also causes JVM to generate a hs_err_pid log file.

This issue is fixed in JavaFx version 11.0.2. Let's bump JavaFx to the appropriate version to avoid this issue.

damithc reviewed (changes requested):
Commit 1:
The commit message (subject or body) does not specify what is the actual issue being fixed. 
Include a brief description. In addition, if there is an online description of the problem, we can give a link.
`described in JDK-8210411 presents a compatibility issue for users using` is longer than 72 chars.

Commit 2:
Let's not sneak in unrelated changes into a PR. That reduces the searchability of the PR/issue tracker.

damithc reviewed (commented):
I assume you tested this to confirm the fix works (I didn't).

Separately, check our JavaFX guide (https://se-education.org/guides/tutorials/javaFx.html) and the Grade guide to see if they need updating too.

Commit message subject is too generic?

damithc commented:
@zhongfu I remember you did some investigation into this problem earlier. If you are free, your inputs are welcome.

zhongfu reviewed (approved):
Looks good to me; I can confirm that this PR allows AB3 to run on Wayland without needing to force the use of X11 explicitly

(p.s. sorry for the late reply -- I missed this somehow)

damithc commented:
@zhongfu thanks very much for checking. :+1:

Eclipse-Dominator committed:
Fix JavaFx crashing in some linux distributions

Conflict between JavaFx version 11 and Ubuntu 18.04 and above
described in JDK-8210411 will cause JVM to crash for users
using Ubuntu 18.04 and above.
This bug is patched in version 11.0.2
(bugs.openjdk.org/browse/JDK-8210411)

Let's bump JavaFx to version 11.0.2 to avoid JVM crashes.
